modules/scanning-images.adoc

```diff
@@ -142,17 +106,70 @@ Scanner identifies vulnerabilities in images that contain the following Linux di
 | `sles:11`^[2]^, `sles:12`^[2]^, `sles:15`^[2]^, `opensuse-leap:15.0`^[2]^, `opensuse-leap:15.1`^[2]^
 
 | link:http://releases.ubuntu.com/[Ubuntu]
-| `ubuntu:14.04`, `ubuntu:16.04`, `ubuntu:18.04`, `ubuntu:20.04`,`ubuntu:22.04`, `ubuntu:23.10`, `ubuntu:24.04`
+| `ubuntu:14.04`, `ubuntu:16.04`, `ubuntu:18.04`, `ubuntu:20.04`, `ubuntu:22.04`, `ubuntu:24.04`, `ubuntu:24.10`
 
 The following vulnerability sources are not updated by the vendor:
-`ubuntu:12.04`, `ubuntu:12.10`, `ubuntu:13.04`, `ubuntu:14.10`, `ubuntu:15.04`, `ubuntu::15.10`, `ubuntu::16.10`, `ubuntu:17.04`, `ubuntu:17.10`, `ubuntu:18.10`, `ubuntu:19.04`, `ubuntu:19.10`, `ubuntu:20.10`, `ubuntu:21.04`, `ubuntu:21.10`, `ubuntu:22.10`, `ubuntu:23.04`, `debian:8`^[1]^, `debian:9`^[1]^, `debian:10`^[1]^
+`ubuntu:12.04`^[1]^, `ubuntu:12.10`^[1]^, `ubuntu:13.04`^[1]^, `ubuntu:14.10`^[1]^, `ubuntu:15.04`^[1]^, `ubuntu::15.10`^[1]^, `ubuntu::16.10`^[1]^, `ubuntu:17.04`^[1]^, `ubuntu:17.10`^[1]^, `ubuntu:18.10`^[1]^, `ubuntu:19.04`^[1]^, `ubuntu:19.10`^[1]^, `ubuntu:20.10`^[1]^, `ubuntu:21.04`^[1]^, `ubuntu:21.10`^[1]^, `ubuntu:22.10`^[1]^, `ubuntu:23.04`^[1]^, `ubuntu:23.10`^[1]^
 |===
 . Only supported in the StackRox Scanner.
 . Only supported in Scanner V4.
 . Images older than June 2020 are not supported in Scanner V4.
 
 [NOTE]
 ====
-* Scanner does not support the Fedora operating system because Fedora does not maintain a vulnerability database.
+Scanner does not support the Fedora operating system because Fedora does not maintain a vulnerability database.
 However, Scanner still detects language-specific vulnerabilities in Fedora-based images.
 ====
+
+[id="supported-package-formats_{context}"]
+== Supported package formats
+
+Scanner can check for vulnerabilities in images that use the following package formats:
+
+|===
+| Package Format | Package Managers
+
+| apk
+| apk
+
+| dpkg
+| apt, dpkg
+
+| rpm
+| dnf, microdnf, rpm, yum
+|===
+
+[id="supported-programming-languages_{context}"]
+== Supported programming languages
+
+Scanner can check for vulnerabilities in dependencies for the following programming languages:
+
```

Comment:
I wasn't sure what `cols="1,3"` means, so I did `cols="2"` based on other tables I saw in other files. Let me know if I should switch it (and let me know what that means 😄)

Reply:
Yeah, I was trying to make it so that the "Programming Language" column was small, and the "Package Format" column was larger. The `1,3` will make it so that the language column is [1 in relative width and the package format is 3](https://docs.asciidoctor.org/asciidoc/latest/tables/adjust-column-widths/), so it will have more space to display the text without wrapping. Sometimes I have to play around with it to see what it actually looks like when it gets converted though.